Makefile

```diff
@@ -273,14 +273,18 @@ test-nightly-server: build-nightly-server
 
 ### Check Rules: static checks, quality tools ###
 
-check: check-shell-scripts
+check: check-shell-scripts check-yaml
 .PHONY: check
-
 # rule requires shellcheck and find to run
 check-shell-scripts: $(filter $(ALT_BIN)%,$(SHELLCHECK))
 	$(SHELLCHECK) -P tests/ -eSC2181 -fgcc $$(find $(ROOT_DIR) -name "*.sh")
 .PHONY: check-shell-scripts
 
+
+check-yaml: $(filter $(ALT_BIN)%,$(YAMLLINT_CMD))
```

Comment:
This target failed on my machine due to projects yaml files which are not aligned with required standard. Fixed by using the following command: ```find ./ -type f -name '*.yml' -exec yq eval --inplace {} \;```

Consider fixing as part of this PR.

Reply:
Does it fail for you? Or just warnings? In my case `make check-yaml` ran successfully with warnings. Nevertheless it makes sense to fix all warnings in this PR itself.

Reply:
@phlogistonjohn asked me some time ago to ignore the warnings since he is convinced our yamls are all perfectly fine :-)

Reply:
@synarete wrote:
> This target failed on my machine due to projects yaml files which are not aligned with required standard.

for me and in the ci, it succeeds with a few warnings.


Fixed by using the following command: `find ./ -type f -name '*.yml' -exec yq eval --inplace {} \;`
> 
> Consider fixing as part of this PR.

I am reluctant to add fixes for things that do not appear to bw broken to me ;-)

Reply:
Yes, I find yamllint (at least the config in the samba-operator repo) overly picky. I especially have trouble remembering what style of indent it wants for lists. I suggested that @obnoxxx just make a few of the things it finds warnings so we we are alerted to what may not be optimal, but we can ignore it for now.

Another issue is that some of these tools seem to have conflicting ideas of what the optimal yaml formatting is and I don't enjoy having to run one tool to generate yamls and then another to reformat them. That's more of an issue in the operator repo than here, since this repo has no yaml generating tools.